RELEASE/scripts/autoscend.ash

```diff
@@ -2769,6 +2769,9 @@ void auto_begin()
 	backupSetting("logPreferenceChange", "true");
 	backupSetting("logPreferenceChangeFilter", "maximizerMRUList,testudinalTeachings,auto_maximize_current");
 	backupSetting("maximizerMRUSize", 0); // shuts the maximizer spam up!
+
+	string userForbidden = get_property("forbiddenStores");
+	backupSetting("forbiddenStores", userForbidden + ",3408540"); // forbid Dance Police
```

Comment:
maybe have an if that first checks to see if userForbidden contains the text 3408540 and only adding it if it is not already there

Reply:
lol good point 👍

Reply:
reason being is that sometimes on unclean shutdown settings are not restored. Thus each time this is run it will add another copy of ,3408540
also... should it maybe check to see if userForbidden == "" and if so not include the leading comma

Reply:
```suggestion
	if(!contains_text(userForbidden, "3408540")
	{
		if(userForbidden == "")
		{
			backupSetting("forbiddenStores", userForbidden + "3408540"); // forbid Dance Police
		}
		else
		{
			backupSetting("forbiddenStores", userForbidden + ",3408540"); // forbid Dance Police
		}
	}
```